IntroToCSharp/Pages/Lessons/Collections/Arrays.razor

```diff
@@ -0,0 +1,379 @@
+    </CodeBlock>
+
+    <MudText>
+        As you can see we have a string of 7 superheroes within our array that make up the original Justice League! Each variable
+        within the array is wrapped within the curly brackets and seperate by a comma! Without the comma, the data will either return an error
+        or the data will be read as a single variable!
+    </MudText>
+    <MudText>
+        Another important aspect to understand an Array is how their index works. An array's index is the position a variable is after
+        within an array. Rather than starting at 1 and counting up, arrays always start and 0 and count up. For our justiceLeague array
+        this means that "Superman" is at the 0 index of the array or in coding terms:
+    </MudText>
+
+    <CodeBlock>
+        string[] justiceLeague = {"Superman", "Batman", "Aquaman", "Green Lanter", "Flash", "Wonder Woman", "Martian Manhunter"};
+        Console.WriteLine("The first superhero in our array is " + justiceLeague[0]);
+    </CodeBlock>
+
+    <MudText></MudText>
+</Section>
+
+<Section Title="Using Arrays">
+    <CodeBlock>
+        string[] avengers = {"Captian America", "Iron Man", "Hulk", "Black Panther", "Doctor Strange", "Scarlet Widow", "Hawkeye"};
+    </CodeBlock>
+    <MudText>
+        Answer some questions about the example above!
+    </MudText>
+
+    <MultipleChoiceQuestion 
+      IsMultiSelect="false" 
+      Question="What is the index for Doctor Strange?"
+      Explanation=""
+      XP=0
+      GoldAward=100
+      >
+        <CheckBoxOption Text="averngers[2]" IsCorrect="false" />
+        <CheckBoxOption Text="avengers[3]" IsCorrect="false" />
+        <CheckBoxOption Text="avengers[4]" IsCorrect="true" />
+        <CheckBoxOption Text="avengers[5]" IsCorrect="false" />
+    </MultipleChoiceQuestion>
+
+    <MultipleChoiceQuestion 
+      IsMultiSelect="false" 
+      Question="True or False: The values in array cannot be changed after the array is created."
+      XP=30
+      GoldAward=0
+      Explanation="">
+        <CheckBoxOption Text="True" IsCorrect="false" />
+        <CheckBoxOption Text="False" IsCorrect="true" />
+    </MultipleChoiceQuestion>
+
+    <CodeBlock>
+        string[] villians = {"Red Skull" "doctor Doom" "evilbadguy12" "Thanos" "Abomination" "Ronan the Conqueror"};
+    </CodeBlock>
+
+    <MultipleChoiceQuestion IsMultiSelect="false" Question="What is wrong with the above problem?"
+        XP=30
+        GoldAward=0
+        Explanation="">
+        <CheckBoxOption Text="The array is missing commas" IsCorrect="true" />
+        <CheckBoxOption Text="The array has more than one data type" IsCorrect="false" />
+        <CheckBoxOption Text="The array needs to use [] instead of {}" IsCorrect="false" />
+        <CheckBoxOption Text="The array needs a semicolon(;) after Ronan the Conqueror" IsCorrect="false" />
+    </MultipleChoiceQuestion>
+</Section>
+
+<Section Title="Arrays without values">
```

Comment:
Maybe call this section "Instantiating an Array". In this section talk about sizes being immutable.

Reply:
In this section you should talk about default values as well.